### ISSUE TITLE
Solid.js types not being referenced correctly

### ISSUE THREAD

**Issue by aryzing** [macaron-css/macaron#56]
Versions

```
"@macaron-css/core": "1.2.0",
"@macaron-css/solid": "1.4.1",
"@macaron-css/vite": "1.4.3",
```

Error message,

```
Could not find a declaration file for module '@macaron-css/solid'. '/project/node_modules/@macaron-css/solid/dist/index.mjs' implicitly has an 'any' type.
  There are types at '/project/node_modules/@macaron-css/solid/dist/index.d.ts', but this result could not be resolved when respecting package.json "exports". The '@macaron-css/solid' library may need to update its package.json or typings.ts(7016)
```

Emphasis on 

> this result could not be resolved when respecting package.json "exports". The '@macaron-css/solid' library may need to update its package.json or typings.ts

**Comment by Mokshit06** [comment]
Thanks for reporting! This has been fixed in v1.5.1.

**Comment by aryzing** [comment]
Appreciate the lightning fast fix, thanks @Mokshit06. Seems the types are being detected now, although are the types themselves actually working? I'm not getting any intellisense with v1.5.1. Is there something missing here?

[simplescreenrecorder-2024-03-24_19.54.20.webm](https://github.com/macaron-css/macaron/assets/4924405/f9f9cbfc-13b6-4670-9b68-a19eb5f08b91)
